docs/tutorials/projects/guardian.md

```diff
@@ -0,0 +1,850 @@
+---
+title: "Security Guardian"
+linkTitle: "Security Guardian"
+weight: 50
+type: "docs"
+description: "Make a functional guardian with a servo motor, some LEDs, a camera, and the ML and Vision Service to detect people and pets."
+webmSrc: "/tutorials/videos/foam-dart.webm"
+mp4Src: "/tutorials/videos/foam-dart.mp4"
+videoAlt: "A guardian detecting a person or pet."
+images: ["/tutorials/img/guardian/preview.gif"]
+tags: ["camera", "vision", "detector", "python"]
+no_list: true
+# Author: Naomi Pentrel
+---
+
+In the run up to the new Zelda release, I realized you could build a stationary guardian robot with a servo and a camera.
+Adding a bit of [machine learning](/services/ml/), you could then make the guardian detect objects or people or pets and follow them around by rotating its head.
+Luckily, I am not the first one to have the idea to build a guardian and there was already a [brilliant guardian 3D model](https://www.thingiverse.com/thing:2391826) on Thingiverse with space for LEDs and a servo.
+
+In this tutorial, I will walk you through the steps to build your own functional guardian.
+
+{{<video webm_src="../../img/guardian/preview.webm" mp4_src="../../img/guardian/preview.mp4" alt="Guardian robot detects person and rotates head to follow them around">}}
```

Comment:
This video is great-- opt. you could probably add more in intro to introduce video & "sell the project", being like "I built this with xyz components and Viam ML model service etc." rather than just "I realized I could build something like this"!